packages/arb-token-bridge-ui/tests/e2e/specs/withdrawERC20.cy.ts

```diff
@@ -103,7 +103,7 @@ describe('Withdraw ERC20 Token', () => {
         })
 
         context('should show clickable withdraw button', () => {
-          cy.findMoveFundsButton().click()
+          cy.startTransfer({ shouldConfirmInMetamask: false })
```

Comment:
`startTransfer` has wait methods to ensure there are no race conditions. For withdrawals we don't want to confirm in metamask popup because we have our dialogs that pop up before that.